workbench-core/accounts/src/models/costCenter.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+
+ */
+
+interface CostCenter {
+  id?: string;
+  name: string;
+  dependency: string;
+  description: string;
+  createdAt?: string;
+  updatedAt?: string;
+  subnetId: string;
+  vpcId: string;
+  envMgmtRoleArn: string;
+  externalId?: string;
```

Comment:
`externalId` is optional on the Account, so that transfers. `id` isn't created until we go through the save flow. I believe `createdAt` and `updatedAt` come from Dynamo on save.

Reply:
Going to make `id`, `createdAt`, and `updatedAt` required based on the assumption that we only create the whole model when its about to be saved, so all this data should be present.